tor/controller.go

```diff
@@ -81,12 +81,19 @@ type Controller struct {
 
 	// version is the current version of the Tor server.
 	version string
+
+	// The IP address which we tell the Tor server to use to connect to the LND node.
+	// This option optional. It is set through the '--tor.targetipaddress' commandline
```

Comment:
We can omit this line and the one below because this can be used elsewhere without the context of the CLI flag.